next.clayui.com/content/docs/components/tabs.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Tabs"
+description: "Tabs organize similar content together into individual sections in the same page."
+packageNpm: "@clayui/tabs"
+packageVersion: "3.0.0"
+packageStatus: "Stable"
+sibling: "docs/components/css-tabs.html"
+---
+
+import {
+    Tabs, TabsDropdown
+} from '../../../src/components/clay/Tabs';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [Composing](#composing)
+- [API](#api)
+  - [Item](#item)
+  - [Content](#content)
+  - [TabPane](#tabpane)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/tabs/">Tabs Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	We recommend that you review the <a href="https://storybook.clayui.com/?path=/story/components-claycard--claycardwithinfo" target="_blank">use cases in the Storybook</a>.
+</div>
+
+## Composing
+
+`ClayTabs` is a [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html), we left this component uncontrolled because the user can use Dropdowns or another control to select a `TabPane` for render.
+
+For composing a Tab component, use `ClayTabs` component for defining a modern style, described by Lexicon, use `modern` property. If you need to justify the texts inside the `ClayTabs.Item`, use `justified` property.
+
+Use `ClayTabs.Item` for defining a TabPane's trigger. For using this trigger as an `anchor` element, just define `href` property, otherwise, the trigger will be a `button`. Use `ClayTabs.TabPane` for wrapping the content that will be rendered by TabPane as the following example:
+
+<Tabs />
+
+As an uncontrolled component, Tabs allows you to control the `active` TabPane. Thereby, we can select the active TabPane by a Dropdown menu. For example:
```

Comment:
This is still a technically a controlled version since we require the user to interact with `onClick` and `active`

Reply:
https://github.com/liferay/clay/pull/2603#discussion_r332689573